Cargo.lock

```diff
@@ -1741,6 +1741,12 @@ dependencies = [
  "regex",
 ]
 
+[[package]]
+name = "env_home"
+version = "0.1.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "c7f84e12ccf0a7ddc17a6c41c93326024c42920d7ee630d04950e6926645c0fe"
```

Comment:
Seeing this dependency here, are there possible bugs from our own environment handling not updating fully before calling into the `which` crate?